contracts/Uniswap/UniswapConfig.sol

```diff
@@ -711,6 +764,20 @@ contract UniswapConfig {
         return getTokenConfigBySymbolHash(keccak256(abi.encodePacked(symbol)));
     }
 
+    /**
+     * @notice Get the config for the reporter
+     * @param reporter The address of the reporter of the config to get
+     * @return The config object
+     */
+    function getTokenConfigByReporter(address reporter) public view returns (TokenConfig memory) {
```

Comment:
I'm wondering whether it would be clearer to use `aggregator` instead of `reporter`. Thoughts?

Reply:
`reporter` was kept for the sake of using terms and concepts familiar to the Compound community. I think I'd rather keep as is, since it would change the naming conventions in a few other places like `invalidateReporter()`.

Reply:
Makes sense 👍